### ISSUE TITLE
`Auto upload`: Manual `Remove from camera roll`

### ISSUE THREAD

**Issue by jtagcat** [nextcloud/ios#1590]
### Actual behaviour

Currently, after enabling `Auto upload`, and `Remove from camera roll`, the user gets annoyed with deletion requests all the time.

![image](https://user-images.githubusercontent.com/38327267/116982234-128b2180-acb8-11eb-892c-c818559f8647.png)

Every time the app is opened, a somewhat 'dangerous' prompt appears. This might:
 - build up negative association towards Nextcloud.
 - train the user in clicking delete with other apps as well.

Sharing is hindered as well. On some apps, you just can't share photos, videos from Nextcloud.   
On other apps (ex: Telegram) you have to do `Attach` (giving a selection of photos) → `File` → `iCloud Drive` → (if not selected) (discovered by trying hard, accidentally, not obvious) swipe left or top left `Browse` (what can be dismissed as a lookalike to `Back`) → `Nextcloud` → `{upload folder}` → most recent file.

### Expected behaviour

After enabling `Remove from camera roll`, show an option `Prompt to remove on every app startup`  (true=current behaviour; perhaps even disabled by default?).

Probably better: don't have the option at all, show the `Free up space` if auto-prompt is disabled, and educate the user on first use (not first click! if they don't finish the first time, don't dismiss it). (The current `Remove from camera roll` would need to change to `Always/Automatically prompt /every time to/ remove from camera roll`.

Having the option disabled would add an item to the `More` menu:
![image](https://user-images.githubusercontent.com/38327267/116982444-54b46300-acb8-11eb-9832-d270ffdf8721.png)

The `Free up space` / `Remove uploaded from camera roll` button should:
 - not have a trash bin icon, might give mixed/false signals, that the files will be actually deleted.
 - be in the top few. The iPhone SE I have shows only the top 4 — it should be easily accessible.
 - after being pressed, jump straight to the OS / native deletion prompt (unless first time, see above).

### Environment data
**iOS version:** 14.5 (18E199)  
**Nextcloud iOS app version:** 3.5.0 (15)

### Screenshots of other observations
So I explored around, downloaded Google Photos to see how they do it on iOS.

The 'delete from everywhere/cloud' prompt is very lookalike to the native/OS one:  
![image](https://user-images.githubusercontent.com/38327267/116985302-c9d56780-acbb-11eb-9daa-b3d1223c9737.png)

**Comment by marinofaggiana** [comment]
@jtagcat thanks for your consideration but the view of delete photo is manage directly from iOS and appear automatically so I can't change nothing of that ...

**Comment by jtagcat** [comment]
Yeah, I don't have any complaints about that. The issue is about (the solution of) not automatically evoking the deletion, having a button to evoke the dialogue.
